cloudsim-plus-testbeds/src/main/java/org/cloudsimplus/migration/ExamplePowerVmAllocationPolicyMigrationStaticThreshold.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.cloudsimplus.migration;
+
+import java.io.FileNotFoundException;
+import java.util.List;
+import org.cloudbus.cloudsim.allocationpolicies.power.PowerVmAllocationPolicyMigrationAbstract;
+import org.cloudbus.cloudsim.hosts.power.PowerHost;
+import org.cloudbus.cloudsim.selectionpolicies.power.PowerVmSelectionPolicy;
+import org.cloudsimplus.sla.readJsonFile.SlaMetric;
+import org.cloudsimplus.sla.readJsonFile.SlaReader;
+
+/**
+ *
+ * @author raysaoliveira
+ */
+public class ExamplePowerVmAllocationPolicyMigrationStaticThreshold extends PowerVmAllocationPolicyMigrationAbstract {
+
+    /**
+     * @see #getOverUtilizationThreshold(PowerHost)
+     */
+    private double overUtilizationThreshold;
+
+    /**
+     * Creates a PowerVmAllocationPolicyMigrationStaticThreshold.
+     *
+     * @param vmSelectionPolicy    the policy that defines how VMs are selected for migration
+     * @param overUtilizationThreshold the over utilization threshold
+     */
+    public ExamplePowerVmAllocationPolicyMigrationStaticThreshold(
+        PowerVmSelectionPolicy vmSelectionPolicy,
+        double overUtilizationThreshold) {
+        super(vmSelectionPolicy);
+        setOverUtilizationThreshold(overUtilizationThreshold);
+    }
+    
+    private void VerifyCpuMetricMaxThreshold() throws FileNotFoundException {
+
+        String METRICS_FILE = "/Users/raysaoliveira/Desktop/TeseMestradoEngInformatica/cloudsim-plus/cloudsim-plus-testbeds/src/main/java/org/cloudsimplus/sla/readJsonFile/SlaMetrics.json";
+        SlaReader reader = new SlaReader(METRICS_FILE);
+        List<SlaMetric> metrics = reader.getContract().getMetrics();
+        metrics.stream()
+                .filter(m -> m.isCpuUtilization())
+                .findFirst()
+                .ifPresent(this::catchCpuOverUtilization);
+    }
+
+    private void catchCpuOverUtilization(SlaMetric metric) {
```

Comment:
The name of the method should be setCpuOverUtilizationFromSlaContract.
It should receive a SlaContract and find the required metric inside it.
In fact this method should be moved to your example class, since this class have to be deleted as mentioned in the previous comment.